core/core-db-repositories/src/main/resources/db/migration/mysql/V1_13__Notification_sending_priority.sql

```diff
@@ -0,0 +1,9 @@
+alter table notification
+    add column sending_priority varchar(50) null;
+
+update notification
```

Comment:
This one, maybe will cause issues on the container startup, for big projects (fintech) for example

Reply:
Will try to test the migrations for MySQL with generated data.